### ISSUE TITLE
val_loss and val_acc not correct for Resnet50 model during training?

### ISSUE THREAD

**Issue by Razza99** [Razza99/Flowers#1]
Training the initial Resnet50 transfer model shows train accuracy increase to 80%, however validation accuracy always stays at random chance (20%).
This issue remains even when test set = train set...

When manually checked, the accuracy on test set is actually around 76%

The same training pipeline for VGG16 transfer model seems to work fine however.
